src/06-loops.js

```diff
@@ -11,8 +11,12 @@
  *  //> 25
  *  (3 + 4 + 5 + 6 + 7 = 25)
  */
-function rangeSum(min, max) {}
-
+function rangeSum(min, max) {
+  let sumOf = {min,max}
```

Comment:
Be careful here on line 15.  We are actually using objet shorthand to create an object that looks like this: ` { min: min, max: max} ` when I think what we want is an array.